<launch>
    <!-- 参数配置 -->
    <arg name="ugv_namespace" default="ugv_0" />
    <arg name="input_z" default="0.0" />
    
    <!-- 坐标转换节点 -->
    <node pkg="lstm_land" type="coord_converter.py" name="uwb_coord_converter" output="screen">
        <param name="ugv_namespace" value="$(arg ugv_namespace)" />
        <param name="default_z" value="$(arg input_z)" />
    </node>

    <!-- 可视化节点 -->
    <!-- <node pkg="rviz" type="rviz" name="rviz" args="-d $(find your_pkg)/config/display.rviz"/> -->
</launch>